apps/site/assets/ts/stop/components/DepartureList.tsx

```diff
@@ -37,14 +37,19 @@ const DepartureList = ({
   alerts,
   targetDate
 }: DepartureListProps): ReactElement<HTMLElement> => {
-  const tripForSelectedRoutePattern: Trip | undefined = departures[0]?.trip;
   const isCR = isACommuterRailRoute(route);
 
   // don's show cancelled departures for subway
   const modeSpecificDepartures: DepartureInfo[] = filter(
     departures,
     (d: DepartureInfo) => !(d.isCancelled && d.routeMode === SUBWAY)
   );
+  const alertsShouldSuppressDepartures =
+    alerts.some(alert => {
+      return isCurrentLifecycle(alert) && alert.effect === "shuttle";
+    }) || hasSuspension(alerts);
```

Comment:
I wonder if suspension alerts also need to be checked if current lifecycle 🤔

Reply:
Would not be surprising 😩